README.md

```diff
@@ -1,47 +1,49 @@
-## Hashgraph application template
+# Hashgraph Application Template
 This is an alternative (arguably friendlier) maven setup for Hashgraph SDK based applications.
 
-It allows you to build and run hashgraph apps using the SDK more like an external resource rather than creating new modules along the demo apps inside the SDK directory structure.
+It allows you to build and run Hashgraph apps using the SDK more like an external resource rather than creating new modules along the demo apps inside the SDK directory structure.
 
-Clone this repo to whatever location (independend from the Hashgraph sdk).
-#### Install Hashgraph SDK
-Follow the pre-requisites, download and unzip the SDK to preferred location. See https://dev.hashgraph.com/docs/installation/
+Clone this repo to whatever location (independent from the Hashgraph sdk).
+## Install Hashgraph SDK
+Follow the pre-requisites, download and unzip the SDK to preferred location. See https://www.swirlds.com/download/
```

Comment:
I think the previous url (https://dev.hashgraph.com/docs/installation/) is a better fit as it describes the prerequisites for installing the Hashgraph SDK (e.g. JCE) which are not reiterated here. That page also includes the download location for the sdk itself.